cmd/pctl/install.go

```diff
@@ -8,15 +8,17 @@ import (
 	"github.com/urfave/cli/v2"
 
 	"github.com/weaveworks/pctl/pkg/catalog"
+	"github.com/weaveworks/pctl/pkg/client"
 	"github.com/weaveworks/pctl/pkg/git"
 	"github.com/weaveworks/pctl/pkg/runner"
 )
 
 func installCmd() *cli.Command {
 	return &cli.Command{
-		Name:      "install",
-		Usage:     "generate a profile subscription for a profile in a catalog",
-		UsageText: "pctl --catalog-url <URL> install --subscription-name pctl-profile --namespace default --branch main --config-secret configmap-name <CATALOG>/<PROFILE>[/<VERSION>]",
+		Name:  "install",
+		Usage: "generate a profile subscription for a profile in a catalog",
+		UsageText: "pctl --catalog-url <URL> install --subscription-name pctl-profile --namespace default --profile-branch main --config-secret configmap-name <CATALOG>/<PROFILE>[/<VERSION>]\n   " +
+			"pctl install --subscription-name pctl-profile --namespace default --profile-branch development --profile-url https://github.com/weaveworks/profiles-examples --profile-path bitnami-nginx",
```

Comment:
shall we add some more comments? E.g. `to install from profile catalog entry...` , `to install directly from a profile file..`

Reply:
sure 👍